docker/actions-secret-entrypoint.sh

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env bash
 set -euo pipefail
 args=("$@")
-exec /app/docker/entrypoint.sh ggshield secret scan -v ${args[@]} ci
+ggshield secret scan -v ${args[@]} ci
```

Comment:
You can keep the `exec` here, it's a small optimization that makes `ggshield` replace the bash interpreter.